modules/check_in/app/controllers/check_in/application_controller.rb

```diff
@@ -25,6 +25,22 @@ def additional_logging?
       Flipper.enabled?('check_in_experience_logging_enabled')
     end
 
+    def minimal_auth_token
```

Comment:
we seem to be using the terms minimal auth and low auth interchangeably. should we standardize on one of them, maybe low auth?

Reply:
Maybe, I will make it minimal auth for the feature flag.  If we change everything to low auth, then the cookies object having `min_session` will be out of sync.  

At the same time, not inlined in using low_auth_session in cookie object as it is visible to the client. Please let me know.

Reply:
As discussed offline, modified the method name and cookie name in the latest commit